src/rviz_visual_tools.cpp

```diff
@@ -222,7 +222,7 @@ bool RvizVisualTools::loadRvizMarkers()
   // ID
   sphere_marker_.ns = "Sphere";
   // Set the marker type.
-  sphere_marker_.type = visualization_msgs::Marker::SPHERE_LIST;
+  sphere_marker_.type = visualization_msgs::Marker::SPHERE;
```

Comment:
It should not, since there are separate `sphere_marker_` and `spheres_marker_` instances of `visualization_msgs::Marker`.